src/index/utxo_entry.rs

```diff
@@ -21,13 +21,12 @@ enum Sats<'a> {
 /// by that many 11-byte sat range entries, otherwise the total output value
 /// stored as a varint.
 ///
-/// If `--index-addresses`, the script pubkey stored as a varint followed by
-/// that many bytes of data.
-///
 /// If `--index-inscriptions`, the list of inscriptions stored as
 /// `(sequence_number, offset)`, with the sequence number stored as a u32 and
 /// the offset as a varint.
 ///
+/// The script pubkey stored as a varint followed by that many bytes of data.
```

Comment:
Why did this comment move? It looks like the order of how things are stored didn't change.